lib/server.js

```diff
@@ -187,10 +187,10 @@ function Server(options) {
 
     this.router.on('mount', this.emit.bind(this, 'mount'));
 
-    if (!options.handleUpgrades && PROXY_EVENTS.indexOf('upgrade') === -1) {
-        PROXY_EVENTS.push('upgrade');
+    if (!options.handleUpgrades && this.proxyEvents.indexOf('upgrade') === -1) {
```

Comment:
Now that proxyEvents is specific to the instance of the server being created, I don't think we even need the indexOf check right?

Reply:
@DonutEspresso: right, I can remove it.